TrueSDK/Core/TCError.m

```diff
@@ -79,6 +79,26 @@ + (TCError *)errorWithCode:(TCTrueSDKErrorCode)code
                                 userInfo:@{kTrueSDKErrorDescription : errorDescription ? errorDescription : @""}];
 }
 
++ (TCError *)errorWithDictionary: (NSDictionary *)dictionary
+{
+    if (dictionary[@"code"] == nil) {
+        return nil;
+    }
+    
+    NSInteger code = (NSInteger)dictionary[@"code"];
```

Comment:
shouldn't be NSNumber?

Reply:
NSError demands an NSInteger explicitly.